string_assignment.js

```diff
@@ -0,0 +1,66 @@
+//STRING EXERCISES
+
+
+// //1.
+// let myStr = 'You can do it!'
```

Comment:
No need to put "charAt" here. Simple [ ] would be okay. On a side note, Yes! You can do it!